NSS-Day-Cohort-49/tabloid-cli-spiny-crabs
# finished tag add

Code changed: Added Tag Add functionality

To test:

Fetch branch
Navigate to pr-tag-add branch
Run app, navigate to tag management
Select tag add
Add a tag
List tags to see if your tag was added
